pkg/kv/kvclient/kvcoord/dist_sender.go

```diff
@@ -520,8 +626,12 @@ func NewDistSender(cfg DistSenderConfig) *DistSender {
 		ds.latencyFunc = ds.rpcContext.RemoteClocks.Latency
 	}
 
-	if cfg.TestingKnobs.OnRangeSpanningNonTxnalBatch != nil {
```

Comment:
Why did this get removed? I think this caused https://github.com/cockroachdb/cockroach/issues/104865.